source/assets/sass/components/_callouts.scss

```diff
@@ -51,16 +51,6 @@
   }
 }
 
-@include breakpoints.sl-breakpoint--medium {
-  .sl-l-container--overview {
-    font-size: var(--sl-font-size--large);
-
-    .sl-c-callout {
-      font-size: var(--sl-font-size--small);
-    }
-  }
-}
```

Comment:
This `.sl-l-container--overview` class was never used in the site, afaict.

Reply:
![sl-l-container-overview](https://user-images.githubusercontent.com/1581694/230470745-a21bd0d7-9bc1-4fdb-b155-109e09c8025b.jpg)

@jgerigmeyer I see it in the old source on pages with both sidebars, but I don't see where it may be actually applied?.?.

Reply:
@stacyk Correct -- I should've said I don't think it was being _applied_ anywhere. See related comment: https://github.com/oddbird/sass-site/pull/33#discussion_r1159990209

Reply:
I don't think there's anything remaining to update? I removed the unused styles, and also didn't re-implement the unused class.

Reply:
sorry, yeah, I didn't see the second class was indented at first... you're right.